packages/core-data/src/entities.js

```diff
@@ -40,6 +41,22 @@ async function loadPostTypeEntities() {
 	} );
 }
 
+/**
+ * Returns the list of the taxonomies entities.
```

Comment:
Nit: The function returns a promise. The promise resolves to a list of the taxonomies entities.

Aside: This is probably one of the only things I dislike about `async` / `await`, that it becomes very non-obvious that it will always return a Promise (I think even if the function has no `return` statement? My uncertainty reenforces my point 😄 ).